### PR TITLE
Docs 0.1.0dev

### PR DESCRIPTION
**Pull request by nvnieuwk**
To create the docs, install `mkdocs` with `pip install mkdocs && pip install mkdoc-material`

Run `mkdocs serve` to create a test web page and go to http://127.0.0.1:8000/structural/ to see your web page

Mkdocs documentation: https://www.mkdocs.org/
Mkdocs material documentation: https://squidfunk.github.io/mkdocs-material/reference/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,62 +10,6 @@
 
 ## Introduction
 
-**nf-cmgg/structural** is a bioinformatics best-practice analysis pipeline for calling structural variants from short reads.
+**nf-cmgg/structural** is a bioinformatics best-practice analysis pipeline for calling SVs (structural variants), CNVs (copy number variants) and repeat region expansions from short DNA reads. The pipeline handles the calling of the variants and postprocessing (filtering, annotating...)
```
Comment (mvheetve) [inline README.md:13]: structural variants (SVs), copy number variants (CNVs) and repeat region expansions (RREs)

File: docs/index.md [hunk 2]
```diff
@@ -1,10 +1,56 @@
 ---
 title: nf-cmgg/structural
-description: A bioinformatics best-practice analysis pipeline for calling structural variants from short reads.
+description: A bioinformatics best-practice analysis pipeline for calling SVs (structural variants), CNVs (copy number variants) and repeat region expansions from short DNA reads
```
Comment (mvheetve) [inline docs/index.md:3]: structural variants (SVs), copy number variants (CNVs) and repeat region expansions (RREs)

File: docs/index.md [hunk 3]
```diff
@@ -1,10 +1,56 @@
...
 ---
 
---8<-- "README.md:10:16"
+## Introduction
+
+**nf-cmgg/structural** is a bioinformatics best-practice analysis pipeline for calling SVs (structural variants), CNVs (copy number variants) and repeat region expansions from short DNA reads. The pipeline handles the calling of the variants and postprocessing (filtering, annotating...)
```
Comment (mvheetve) [inline docs/index.md:8]: just use abbreviations, you have declared them higher up

**Review by mvheetve — APPROVED** [review]
just a few comments, but fine by me!

File: docs/index.md (hunk 3 above)
Comment (nvnieuwk) [reply docs/index.md:8]: This is the start of the documentation site, so I think it should be fully shown here too

**Pull request merged**